main.go

```diff
@@ -96,6 +97,10 @@ func SetLogger(logger log.Logger) func(delegate http.Handler) http.Handler {
 	}
 }
 
+func GetLogger(ctx context.Context) bascule.Logger {
+	return log.With(logging.GetLogger(ctx), "ts", log.DefaultTimestampUTC, "caller", log.DefaultCaller)
```

Comment:
Make sure that you've set a logger in the context with `logging.WithLogger`.  Otherwise, you'll get `logging.DefaultLogger()`, which logs to `os.Stdout`.  In turn, `stdout` might be piped to `/dev/null` depending on how you start your process.  Check for a `console.out` file.